pkgs/os-specific/linux/esdm-es/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub, kernel, kmod, esdm }:
+
+stdenv.mkDerivation rec {
+  pname = "esdm-es";
+  version = "0.6.0";
```

Comment:
```suggestion
  version = "${esdm.version}";
```

Reply:
Incorporated this change in commit https://github.com/NixOS/nixpkgs/pull/238777/commits/fdd5a583debf3198a7e994d13cb5bbced87a7db0